cleanlab/cleanlab-studio
# externally hosted media upload support

### Description
CLI changes to support uploading externally hosted media datasets.

### How should this be tested?
- Pull changes from `feat/external-media-upload` branch in `cleanlab-studio` and `cleanlab-studio-backend` repos
- Run studio app locally
- Try uploading dataset containing image urls from CLI (example: [dog_breeds_img_url_dataset.csv](https://github.com/cleanlab/cleanlab-studio/files/10798985/dog_breeds_img_url_dataset.csv))
- Make sure other image upload formats still work from CLI (simple image upload, csv manifest upload)
- Check if datasets were uploaded successfully in frontend (will need to pull from `feat/external-media-upload` branch in frontend repo for images to render properly when viewing project)
